### DISCUSSION TITLE
Material for "Case Studies"

### DISCUSSION THREAD

**Discussion by yanodd** [becarpenter/book6#64] [Ideas]
Hello Brian, all

What did you have in mind for the "Case Studies" section? For example, would a case study about an ISP's "IPv6-only (MAP-E) residential service" be of interest?

Regards,
Yannis

**Comment by buraglio** [comment]
I believe it would be. Although service provider networks are a bit further along, I believe all standard and supportable case studies should be considered for inclusion. I have definitely seen and spoken to ISPs that are still trying to figure out their strategies, and the more succinct examples they can find, the better.

**Comment by becarpenter** [comment]
@yanodd, "What did you have in mind for the "Case Studies" section?" To be honest I don't have a general answer to that question, except that each case should be reasonably short, and obviously relevant to a whole class of networks. So I think your suggestion would be a very good start for this chapter. Please start writing! The [Chapter Template](https://github.com/becarpenter/book6/blob/main/99.%20Chapter%20Template/99.%20Chapter%20Template.md) is your friend.

**Comment by yanodd** [comment]
OK, I'll draft something for all to see and comment :)

**Comment by becarpenter** [comment]
Ping?

**Comment by yanodd** [comment]
Sorry, just taking some downtime right now but I definitely will get on with this
